public/css/main.css

```diff
@@ -1,15 +1,22 @@
-html,
-body {
+html {
```

Comment:
All of these CSS changes will affect the overall game. We will have to review these changes together to see if this is best to make these changes.